on_startup.py

```diff
@@ -1,37 +1,69 @@
-import glob
-import json
-import logging
+import os
 import requests
 import subprocess
-from requests.exceptions import RequestException
 
 import django
+
+from requests.exceptions import RequestException
+
 from django.core.management import call_command
 
-logger = logging.getLogger(__name__)
+HIGLASS_DATA_TYPE = "higlass_data_type"
+HIGLASS_FILE_TYPE = "higlass_file_type"
+
+TILE_SETS = {}
+
+
+def get_refinery_input_as_json():
+    return requests.get(os.environ["INPUT_JSON_URL"]).json()
```

Comment:
Generally, we don't want to litter the code with error checking, but this might be an ok place, so we know there are other modes, and better feedback might be valuable... maybe getattr and then make sure it's not None?